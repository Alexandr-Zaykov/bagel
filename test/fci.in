Molecule: C1
  Atom(H,  0.0, 0.0, 0.0)
  Atom(F,  0.0, 0.0,-1.0)

Basis:
  6-31g

FCI:
