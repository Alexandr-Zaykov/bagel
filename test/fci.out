
  --------------------------------------------------
  This is the package       ** Poly-R12 **          
  written by Toru Shiozaki (shiozaki.toru@gmail.com)
  --------------------------------------------------

  === Geometry ===

  Symmetry: c1

  H   0.0000000000   0.0000000000   0.0000000000
  F   0.0000000000   0.0000000000  -1.0000000000

  ---------------------------
        FCI calculation      
  ---------------------------

  Performs exactly the same way as Knowles & Handy 1984 CPL

  o lexical mappings
  o alpha-beta strings
      length:           924
  o single displacement lists (alpha)
      length:         25410
  o single displacement lists (beta)
      length:         25410

  === Nuclear Repulsion===
  
     9.0000000000

  === RHF iteration (6-31g)===
  
      0    -97.372507065313      0.243750247403           0.00
      1    -98.583681950546      0.168098929538           0.00
      2    -98.766106060756      0.028227124051           0.00
      3    -99.960135695480      0.062538288737           0.00
      4    -99.824884672090      0.041725840813           0.00
      5    -99.492349131489      0.004978819128           0.00
      6    -99.438088321303      0.001008777244           0.00
      7    -99.433462015797      0.000596180049           0.00
      8    -99.433889970396      0.000287718180           0.00
      9    -99.434321667400      0.000021579954           0.00
     10    -99.434416593575      0.000001116302           0.00
     11    -99.434416485993      0.000000038334           0.00
     12    -99.434416477866      0.000000002368           0.00
     13    -99.434416477833      0.000000000132           0.00
     14    -99.434416477853      0.000000000032           0.00
     15    -99.434416477853      0.000000000002           0.00
     16    -99.434416477854      0.000000000000           0.00
  
    * SCF iteration converged.

  - AO integrals are computed and stored in core

  === FCI iteration ===

      0    0    -99.434416477853      0.758148867611           0.54
      1    0    -99.538482806131      0.207155797576           0.54
      2    0    -99.543722212411      0.050831723248           0.54
      3    0    -99.544048507186      0.011396998741           0.55
      4    0    -99.544065953197      0.002522813350           0.55
      5    0    -99.544066643878      0.000565763114           0.55
      6    0    -99.544066681694      0.000165127359           0.56
      7    0    -99.544066685566      0.000043654093           0.57
      8    0    -99.544066685816      0.000013123338           0.60
      9    0    -99.544066685837      0.000003242325           0.62
     10    0    -99.544066685838      0.000000681863           0.63
     11    0    -99.544066685838      0.000000133660           0.64
     12    0    -99.544066685838      0.000000032855           0.64
     13    0    -99.544066685838      0.000000006036           0.60

  Normal termination.
  --------------------------------------------------

